docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs

```diff
@@ -0,0 +1,265 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.ML;
+using Microsoft.ML.Transforms;
+using static Microsoft.ML.DataOperationsCatalog;
+using System.Linq;
+using Microsoft.ML.Data;
+using System.IO.Compression;
+using System.Threading;
+using System.Net;
+
+namespace Samples.Dynamic
+{
+    public class ResnetV2101TransferLearningTrainTestSplit
+    {
+        public static void Example()
+        {
+            string assetsRelativePath = @"../../../assets";
+            string assetsPath = GetAbsolutePath(assetsRelativePath);
+
+            var outputMlNetModelFilePath = Path.Combine(assetsPath, "outputs", "imageClassifier.zip");
+
+            string imagesDownloadFolderPath = Path.Combine(assetsPath, "inputs", "images");
+
+            //Download the image set and unzip
+            string finalImagesFolderName = DownloadImageSet(imagesDownloadFolderPath);
+            string fullImagesetFolderPath = Path.Combine(imagesDownloadFolderPath, finalImagesFolderName);
+
+            try
+            {
+
+                MLContext mlContext = new MLContext(seed: 1);
+
+                //Load all the original images info
+                IEnumerable<ImageData> images = LoadImagesFromDirectory(folder: fullImagesetFolderPath, useFolderNameasLabel: true);
+                IDataView shuffledFullImagesDataset = mlContext.Data.ShuffleRows(mlContext.Data.LoadFromEnumerable(images));
+                shuffledFullImagesDataset = mlContext.Transforms.Conversion.MapValueToKey("Label")
+                    .Fit(shuffledFullImagesDataset)
+                    .Transform(shuffledFullImagesDataset);
+
+                // Split the data 90:10 into train and test sets, train and evaluate.
+                TrainTestData trainTestData = mlContext.Data.TrainTestSplit(shuffledFullImagesDataset, testFraction: 0.1, seed: 1);
+                IDataView trainDataset = trainTestData.TrainSet;
+                IDataView testDataset = trainTestData.TestSet;
+
+                var pipeline = mlContext.Model.ImageClassification("ImagePath", "Label",
+                            arch: ImageClassificationEstimator.Architecture.ResnetV2101,
+                            epoch: 50,
+                            batchSize: 10,
+                            learningRate: 0.01f,
+                            metricsCallback: (metrics) => Console.WriteLine(metrics),
+                            validationSet: testDataset);
+
+
+                Console.WriteLine("*** Training the image classification model with DNN Transfer Learning on top of the selected pre-trained model/architecture ***");
+
+                // Measuring training time
+                var watch = System.Diagnostics.Stopwatch.StartNew();
```

Comment:
Measuring the time with watch makes the code a bit more verbose or difficult to read.
But to me it is useful, so we can keep it here, too. 

I'm removing it from the samples repo so the code is simpler, though.